frontend/src/lib/mocks/mocks.ts

```diff
@@ -62,3 +67,35 @@ export function mockEndpoints(
     ...overrides,
   }
 }
+
+export class MockMetricsManager implements MetricsManager {
```

Comment:
I think we probably want to use mock functions here. (Also: no need to create a constructor that takes a sessionInfo instance if nothing in the mock uses it.)

```typescript
export class MockMetricsManager {
  public enqueue = jest.fn()
  public incrementDeltaCounter = jest.fn()
  // ... etc
}
```